client/js/controllers/game.js

```diff
@@ -1,177 +1,215 @@
-          } else if (game.curQuestion.numAnswers === 2 &&
-            $scope.pickedCards.length === 2) {
-            // delay and send
-            $scope.hasPickedCards = true;
-            $timeout($scope.sendPickedCards, 300);
+  .controller('GameController', ['$scope', 'game', '$timeout', 'userSearch', '$location', 'MakeAWishFactsService', '$dialog',
+    function GameController($scope, game, $timeout, userSearch, $location, MakeAWishFactsService) {
+      $scope.hasPickedCards = false;
+      $scope.winningCardPicked = false;
+      $scope.showTable = false;
+      $scope.modalShown = false;
+      $scope.game = game;
+      $scope.pickedCards = [];
+      $scope.searchedUsers = [];
+      $scope.invitedUsers = [];
+      $scope.selectedUser = '';
+      let makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
+      $scope.makeAWishFact = makeAWishFacts.pop();
+
+      $scope.pickCard = (card) => {
+        if (!$scope.hasPickedCards) {
+          if ($scope.pickedCards.indexOf(card.id) < 0) {
+            $scope.pickedCards.push(card.id);
+            if (game.curQuestion.numAnswers === 1) {
+              $scope.sendPickedCards();
+              $scope.hasPickedCards = true;
+            } else if (game.curQuestion.numAnswers === 2 &&
+              $scope.pickedCards.length === 2) {
+              // delay and send
+              $scope.hasPickedCards = true;
+              $timeout($scope.sendPickedCards, 300);
+            }
+          } else {
+            $scope.pickedCards.pop();
           }
-        } else {
-          $scope.pickedCards.pop();
         }
-      }
-    };
-
-    $scope.pointerCursorStyle = function() {
-      if ($scope.isCzar() && $scope.game.state === 'waiting for czar to decide') {
-        return { cursor: 'pointer' };
-      }
-      return {};
-    };
+      };
 
-    $scope.sendPickedCards = function() {
-      game.pickCards($scope.pickedCards);
-      $scope.showTable = true;
-    };
-
-    $scope.cardIsFirstSelected = function(card) {
-      if (game.curQuestion.numAnswers > 1) {
-        return card === $scope.pickedCards[0];
-      }
-      return false;
-    };
+      $scope.selectUser = (selectedEmail) => {
+        if ($scope.invitedUsers.length <= 11) {
+          // console.log(_.indexOf($scope.invitedUsers, selectedEmail));
+          if (_.indexOf($scope.invitedUsers, selectedEmail) === -1) {// eslint-disable-line
+            $scope.invitedUsers.push(selectedEmail);
+          }
 
-    $scope.cardIsSecondSelected = function(card) {
-      if (game.curQuestion.numAnswers > 1) {
-        return card === $scope.pickedCards[1];
-      }
-      return false;
-    };
+          const mailObject = {
+            To: selectedEmail,
+            Link: document.URL
+          };
+          console.log(mailObject);// eslint-disable-line
+          userSearch.sendInvite(mailObject).then((data) => {
+            console.log(data.data);// eslint-disable-line
```

Comment:
remove console, replace with success feedback to user, and push user email to invited users, so you can disable invite button